src/Components/CreateImageWizard/CreateImageWizard.scss

```diff
@@ -94,8 +94,13 @@ div.pf-v5-c-alert.pf-m-inline.pf-m-plain.pf-m-warning {
     Temporarily commenting out this code, until we figure out how the handle
     this bug.
     */
-    // flex: 1 1 0
-    
-    // TO DO: Remove after the fix
-    min-height: 800px;
+    flex: 1 1 auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
 }
+
+.pf-v5-c-wizard__footer {
+    position: sticky;
+    bottom: 0;
+}
```

Comment:
You're missing empty line at the end of the line, can you please add it for consitency? :)

Reply:
Whoops! I wouldn't want to mess with consistency. 😅